SonarQube.Common/FilePropertiesProvider.cs

```diff
@@ -24,11 +24,6 @@ public class FilePropertiesProvider : ISonarPropertyProvider
         /// </summary>
         private IDictionary<string, string> properties;
 
-        /// <summary>
-        /// Property name comparisons are be case-sensitive
-        /// </summary>
-        private static StringComparer PropertyKeyComparer = StringComparer.InvariantCulture;
```

Comment:
It's likely that values will be file paths etc which we wouldn't want to compare using the current culture.